tests/pyspark/test_pyspark_model.py

```diff
@@ -313,6 +313,19 @@ class Schema(DataFrameModel):  # pylint:disable=missing-class-docstring
     assert not schema.columns["b"].required
     assert not schema.columns["c"].required
 
+    data = [("5", "b"), ("15", "b")]
+    spark_schema = T.StructType(
+        [
+            T.StructField("a", T.StringType(), False),
+            T.StructField("b", T.StringType(), False),
+            # 'c' column is not required
+        ],
+    )
+
+    df = spark_df(spark, data, spark_schema)
+    df_out = Schema.validate(check_obj=df)
+    assert df_out.pandera.errors == {}
+
```

Comment:
Deepening the test case to ensure that optional fields go through a `.validate()`.

Reply:
Accepted approach, tks for the suggestion.